frontend/src/shared/Data/AppleWatch/index.ts

```diff
@@ -0,0 +1,51 @@
+import axios, { AxiosPromise } from "axios";
+
+const DEV =
+  "http://138.197.134.112:8080/BeaplabEngine-0.0.1-SNAPSHOT/rest/beapengine/applewatch";
+// const PROD = "/api/rest/beapengine/applewatch";
+
+const $axios = axios.create({
+  baseURL: DEV,
+  withCredentials: true,
+  responseType: "json",
+});
+
+export default class AppleWatchService {
+  // constructor() {}
+
+  upload(form: FormData): AxiosPromise {
+    return $axios.post("/upload", form, {
```

Comment:
The correct api route is this
```suggestion
    return $axios.post("/applewatch/upload", form, {
```
Or this if you want to combine fitbit and applewatch together, we can pass a type defining if it is applewatch or fitbit,
```suggestion
    return $axios.post(`/${type}/upload`, form, {
```